src/resources/PowerShellWorkerStrings.resx

```diff
@@ -194,16 +194,16 @@
     <value>Version specification '{0}' is not in the correct format. Please specify the exact version or use the following notation: '{1}'</value>
   </data>
   <data name="FailToInstallFuncAppDependencies" xml:space="preserve">
-    <value>Fail to install FunctionApp dependencies. Error: '{0}'</value>
+    <value>Failed to install function app dependencies. Error: '{0}'</value>
   </data>
   <data name="FailToResolveHomeDirectory" xml:space="preserve">
-    <value>Fail to resolve '{0}' path in App Service.</value>
+    <value>Failed to resolve '{0}' path in App Service.</value>
   </data>
   <data name="InstallingFunctionAppDependentModules" xml:space="preserve">
-    <value>Installing FunctionApp dependent modules to '{0}'.</value>
+    <value>Installing function app dependent modules to '{0}'.</value>
   </data>
   <data name="FailToClenupModuleDestinationPath" xml:space="preserve">
```

Comment:
I don't think this resource string is being used anymore. Please remove it.

Reply:
I can actually see it being used: https://github.com/Azure/azure-functions-powershell-worker/blob/0be9d316c580c8a9d95345b5d6432fb5478d7835/src/DependencyManagement/DependencySnapshotInstaller.cs#L47

Reply:
The message needs to be fixed though: these are not _dependent_ modules, it's the other way around.

Reply:
> The message needs to be fixed though: these are not dependent modules, it's the other way around.

You are right. We should fix the message.

The string I was referring to remove is ```FailToClenupModuleDestinationPath```.

Reply:
Yes. Removing.

Reply:
Thank you.